icicle-compiler/src/Icicle/Sea/FromAvalanche/State.hs

```diff
@@ -220,10 +221,15 @@ defOfFactField (name, ty)
 
 ------------------------------------------------------------------------
 
-defOfResumable :: (Text, ValType) -> Doc
-defOfResumable (n, t)
- =  defOfVar 0 BoolT (pretty hasPrefix <> pretty n) <> semi <> line
- <> defOfVar 0 t     (pretty resPrefix <> pretty n) <> semi
+defOfResumable :: SeaProgramCompute -> (Text, ValType) -> Doc
+defOfResumable compute (n, t)
+ =  defOfVar 0 BoolT (pretty hasPrefix <> nameOfResumable compute (pretty n)) <> semi <> line
+ <> defOfVar 0 t     (pretty resPrefix <> nameOfResumable compute (pretty n)) <> semi
+
+nameOfResumable :: SeaProgramCompute -> Doc -> Doc
+nameOfResumable compute n
+ = let (i,j) = stateComputeName compute
+   in  pretty i <> "_" <> pretty j <> "_" <> n
```

Comment:
I renamed the resumables to have the index of the attribute and compute kernel. the attribute is superfluous but it was already there.
so the attribute info now has something like:
```
ibool_t has_0_1_fridge;
istring_t res_0_1_fridge;
```
instead of just
```
ibool_t has_fridge;
istring_t res_fridge;
```

I didn't rename the outputs because I believe they should be unique. is that correct?